README.md

```diff
@@ -49,15 +49,36 @@ Or alternatively customize your analysis by listing the analysers you want enabl
 
 The `filter` option allows the field to be used as a filter at search time.
 
-You can optionally provide field weightings using the `factor` option or document-level boosts using the `boost` declaration. While these are query options, Stella allows for their static declaration in the `searchable` block for simplicity - they will be applied at query time by default when using `#stella_search`.
+You can optionally provide field weightings to be applied at search time using the `factor` option. These are multipliers. 
+
+Document-level boosts can be applied with the `boost` declaration, see the [field_value_factor](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/query-dsl-function-score-query.html#function-field-value-factor) documentation for boost options. 
+
+While `filter`, `boost` and `factor` are query options, Stella allows for their static declaration in the `searchable` block for simplicity - they will be applied at query time by default when using `#stella_search`.
+
+The searchable block takes a `settings` hash in case you require custom analysis or sharding (see [doc](https://www.elastic.co/guide/en/elasticsearch/guide/current/configuring-analyzers.html)):
+
+```ruby
+
+searchable my_settings do
```

Comment:
it is a bit tricky yes, so a real world example might be overkill for the readme here, but it would be insightful i agree.  I've added one now, let me know if you think it's overcomplicating things. i think it's probably ok.